### ISSUE TITLE
SandboxContext.AccountsAsync() возвращает значение неподходящего типа

### ISSUE THREAD

**Issue by borex99** [TinkoffCreditSystems/invest-openapi-csharp-sdk#103]
Метод `SandboxContext.AccountsAsync()` возвращает значение типа `Task<IReadOnlyCollection<Account>>`, поскольку наследует метод `AccountsAsync()` от класса `Context`. Значения внутри возвращаемого экземпляра верные.

Предполагаю, что правильно было бы переопределить метод `AccountAsync()` в классе `SandboxContext`, который будет возвращать значение типа `Task<IReadOnlyCollection<SandboxAccount>>`, либо отказаться от класса `SandboxAccount`, либо создать преобразование между классами, поскольку метод `SandboxContext.RegisterAsync()` возвращает значение типа `Task<SandboxAccount>`, и нет преобразования между классами `Account` и `SandboxAccount`, хотя их внутреннее содержимое идентичное.

**Comment by rus-art** [comment]
Да, согласен, два идентичных класса. От `SandboxAccount` можно отказаться
